routers/routers.go

```diff
@@ -19,6 +19,9 @@ func SetRoutes() {
 			http.StripPrefix(staticDirectoryPaddedWithSlashes, fileServer))
 	}
 
+	// Root path will redirect to home
+	http.HandleFunc("/", handlers.HandleRootRequest)
```

Comment:
not sure how i feel about it, but we could maybe just name the function something like `redirectGetRequestToHome`?

Reply:
Out of curiosity why have a specific functionality name for this handle request, but not for other ones?